ci/deps/actions-310.yaml

```diff
@@ -56,6 +56,8 @@ dependencies:
   - xlrd>=2.0.1
   - xlsxwriter>=3.0.5
   - zstandard>=0.19.0
+  # TEMP to avoid error when the env gets cryptography >= 42 but older pyopenssl
```

Comment:
If that combination is wrong we should do a repo data patch to fix it. I'll investigate this on the conda-forge side.

Reply:
@ocefpaf seems there is already a PR for this: https://github.com/conda-forge/conda-forge-repodata-patches-feedstock/pull/913

Reply:
Yep. In fact two! https://github.com/conda-forge/conda-forge-repodata-patches-feedstock/pull/918 was just merged and hopefully you don't need this work around.